.cloud-build/execute_changed_notebooks_cli.py

```diff
@@ -47,6 +50,12 @@ def str2bool(v):
     required=False,
     default=100,
 )
+parser.add_argument(
+    "--build_id",
+    type=str,
+    help="The build id (which may be cloud run specified or user explicit.",
```

Comment:
small nit: Maybe clarify what 'cloud run' means to disambiguate it from 'Cloud Run'

Reply:
And what 'user explicit' means.

Reply:
updated